libraries/lib-numeric-formats/NumericConverterFormatter.cpp

```diff
@@ -12,11 +12,8 @@
 
 #include <cmath>
 
-NumericField::NumericField(
-   bool _frac, int _base, int _range, bool _zeropad) noexcept
-    : frac { _frac }
-    , base { _base }
-    , range { _range }
```

Comment:
I'm catching up with this commit now, which did some of what I suggested to simplify `NumericField`.